qemu/tests/kdump_with_stress.py

```diff
@@ -37,21 +38,23 @@ def install_stress_app(session):
         install_cmd = params.get("install_cmd")
 
         logging.info("Fetch package: '%s'" % link)
-        pkg = utils.unmap_url_cache(test.tmpdir, link, md5sum)
-        vm.copy_files_to(pkg, tmp_dir)
+        pkg_name = os.path.basename(link)
+        pkg_path = os.path.join(test.tmpdir, pkg_name)
+        download.get_file(link, pkg_path, hash_expected=md5sum)
```

Comment:
it seems you can only use `md5` or `sha1` here.
```python
    if algorithm not in ['md5', 'sha1']:
        raise ValueError("Unsupported hash algorithm: %s" % algorithm)
```

Reply:
`download.get_file(link, pkg_path) ` is enough.

Reply:
Here, it wants to check whether the package is corrupted or not, so have to give the checksum to the function.

Reply:
sorry, my mistakes.